cmd/openshift/operator/kodata/tekton-addon/addons/01-clustertriggerbindings/gitlab.yaml

```diff
@@ -7,6 +7,8 @@ spec:
   params:
   - name: git-repo-url
     value: $(body.project.git_http_url)
+  - name: git-repo-ssh-url
```

Comment:
this is correct for the mr event according to https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#merge-request-events